src/miner.cpp

```diff
@@ -573,9 +551,14 @@ void GenerateBitcoins(bool fGenerate, CWallet* pwallet, int nThreads)
     if (nThreads == 0 || !fGenerate)
         return;
 
+    CScript coinbaseScript;
+    GetMainSignals().ScriptForMining(coinbaseScript);
```

Comment:
Maybe my backtrace prove (see below) can convince you.
I can also post a linux gdb backtrace or you could add a `sleep(10) in `void CWallet::GetScriptForMining(CScript &script)`.

![bildschirmfoto 2015-06-03 um 11 34 37](https://cloud.githubusercontent.com/assets/178464/7957020/a55ba08e-09e4-11e5-9170-b41acdc98e71.png)